src/test/java/nl/tudelft/contextproject/audio/AudioManagerTest.java

```diff
@@ -39,19 +39,12 @@ public void tearDown() {
 	 */
 	@Test
 	public void testRegisterVolume() {
-		//Register the volume
 		AudioManager.getInstance().registerVolume(an, SoundType.AMBIENT);
-		
-		//Registering should set the volume to the correct type.
 		verify(an, times(1)).setVolume(SoundType.AMBIENT.getGain());
-		
-		//Reset the mock
+
 		reset(an);
-		
-		//Change the gain
+
 		SoundType.AMBIENT.setGain(2.0f);
-		
-		//The volume should have been updated
 		verify(an, times(1)).setVolume(2.0f);
```

Comment:
@Taeir and I can understand each other's preference for these comments, but disagree on wether or not they should be added. Someone else's input would be greatly appreciated.